src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/HashedIdTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeCommitments.Web.Pages.IdentityHashing;
+using SFA.DAS.HashingService;
+using static SAF.DAS.ApprenticeCommitments.Web.UnitTests.EqualityTests;
+
+namespace SAF.DAS.ApprenticeCommitments.Web.UnitTests
+{
+    public class HashedIdTests
+    {
+        private const string alphabet = "abcdefgh12345678";
+        public static IHashingService Hashing = new HashingService(alphabet, "testing");
```

Comment:
It's this test:
https://github.com/SkillsFundingAgency/das-apprentice-commitments-web/blob/20d99c180bd19e063cddea7fddaa38c8f64528f0/src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/EqualityTests.cs#L187-L192

It probably would be clearer if the test actually stated `null == null`, I don't know how easy it would be to do that within the framework of those tests